implementations/rust/ockam/ockam_command/src/status/static/long_about.txt

```diff
@@ -0,0 +1,8 @@
+The status command displays the current status of your machine, including information such as:
+- the current version of Ockam in use
+- the controller (Orchestrator) and project version
+- your spaces and the users associated with them
+- all cryptographic identities stored in your vault and their enrollment status with the Ockam Orchestrator
+- the status of any Nodes, associated routes and protocols, listeners and services, and more
```

Comment:
Let's replace:
- controller (Orchestrator) -> Ockam Orchestrator
- project, space, identity -> capitalized


Also, the list won't look good on the terminal (when running `ockam status --help`). So let's try to summarize the information in a paragraph.